crates/core_arch/src/powerpc/altivec.rs

```diff
@@ -158,32 +158,32 @@ unsafe extern "C" {
     #[link_name = "llvm.ppc.altivec.vmulosh"]
     fn vmulosh(a: vector_signed_short, b: vector_signed_short) -> vector_signed_int;
 
-    #[link_name = "llvm.ppc.altivec.vmaxsb"]
+    #[link_name = "llvm.smax.v16i8"]
```

Comment:
Since these are generic LLVM vector intrinsics, it would make more sense to provide these in `core::intrinsics::simd` and then use them from stdarch, similar to how `simd_neg` is used.

Reply:
I see this is already a pattern in quite a few places in stdarch. This doesn't need to block this PR, it can be resolved separately.

Reply:
yeah, makes sense. I'll add that then.